resources/references/security.md

```diff
@@ -4,10 +4,34 @@
 
 This reference presents a comprehensive compilation of all security measures implemented in Shopware 6, along with instructions on how to configure them.
 
+## ACL in the Administration
+
+The Access Control List (ACL) in Shopware ensures that by default, data can only be created, read, updated, or deleted (CRUD), once the user has specific privileges for a module. [ACL in the Administration](../../concepts/framework/architecture/administration-concept.md#acl-in-the-administration)
+
 ## API aware fields
```

Comment:
Please mention everything either in singular or plural. IMO, singular would be good like "API aware field"